assets/js/components/DataBlock/index.js

```diff
@@ -64,9 +64,9 @@ const DataBlock = ( {
 
 	const handleKeyDown = useCallback(
 		( e ) => {
-			e.preventDefault();
 			if ( 'Enter' === e.key || ' ' === e.key ) {
 				handleClick();
+				e.preventDefault();
```

Comment:
Is the order of this call significant? The IB calls for it to be moved down 1 line (into the condition) but this was moved after the click handler. It doesn't seem to matter but usually the `preventDefault` is called first when needed. If it doesn't matter, let's move it before the `handleClick`, otherwise let's add a comment to explain the significance of the call order.

Reply:
@aaemnnosttv good catch indeed. It does not seem to matter, I'll move it to where the IB dictated.

Reply:
@aaemnnosttv this has been corrected in https://github.com/google/site-kit-wp/pull/7096/commits/b045f29c22647df34732b60805f4601b3b77c0af